### PR TITLE
common/gpu/intel/lunar-lake: enable xe driver

### PR DESCRIPTION
**Pull request by Mic92**
###### Things done

- [ ] Tested the changes in your own NixOS Configuration
- [ ] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

**Comment by Mic92** [comment]
cc @Pandapip1 for review

File: common/gpu/intel/lunar-lake/default.nix [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{ config, lib, ... }:
+{
+  imports = [ ../. ];
+  hardware.intelgpu.driver = lib.mkIf (lib.versionAtLeast config.boot.kernelPackages.kernel.version "6.8") "xe";
+  hardware.intelgpu.vaapiDriver = "intel-media-driver";
```
Comment (Pandapip1) [inline common/gpu/intel/lunar-lake/default.nix:5]:
```suggestion
  hardware.intelgpu = {
    driver = lib.mkIf (lib.versionAtLeast config.boot.kernelPackages.kernel.version "6.8") "xe";
    vaapiDriver = "intel-media-driver";
  };
```

**Review by Pandapip1 — APPROVED** [review]
Diff LGTM.

**Review by Pandapip1 — APPROVED** [review]

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**